up-and-running/auth/oauth-github.md

```diff
@@ -0,0 +1,48 @@
+# GitHub Enterprise OAuth2 Integration
+
+npm Enterprise allows you to log in to your private registry and website using your GitHub Enterprise account.
+
+Follow these steps to set it up:
+
+## Server Configuration
+
+There are two basic steps to server-side configuration: creating an OAuth client in Github Enterprise, and configuring authentication in npm Enterprise.
+
+### Setup OAuth2 client credentials in GitHub Enterprise
+
+1. login to your GitHub Enterprise appliance.
+2. Click on `Settings`, `OAuth Applications`, `Register a new OAuth application`.
```

Comment:
I've been using `**Name**` for buttons or things they should click on-screen. Not super important.